ui/pages/onboarding-flow/create-password/index.scss

```diff
@@ -29,6 +29,18 @@
       }
     }
 
+    &__weak {
```

Comment:
[nit] According to BEM, nesting elements like this (`.foo__bar__baz`) is [discouraged](https://en.bem.info/methodology/faq/#why-not-create-elements-of-elements-block__elem1__elem2). What do you think about placing this as a direct child of `.create-password` instead?

Reply:
@mcmire Fixed in: 31b1950541693058091725c294105bf99a39b4fa